hack/run-test-cleanup.sh

```diff
@@ -126,3 +126,40 @@ while read -r report; do
         fi
     fi
 done <<< "$REPORTS"
+
```

Comment:
This script does no cleanup, it does test output capture, should be renamed, IMO.

Reply:
Yeah I've been wondering why I named that script `run-test-cleanup.sh` for a while now, definitely something that should be renamed to properly reflect what it's doing.